src/components/Slideover/slideover.scss

```diff
@@ -44,5 +44,6 @@
 
   .sb-modal-header__title {
     padding-right: 30px;
+    word-break: break-all;
```

Comment:
Hey, I have some css suggestions for this.
```
.sb-modal-header__title {   
    word-break: break-word; // so we don't break words into characters
    width: calc(100% - 36px); // which is the width of the close icon
    line-height: 36px; // the space between every line of the title is too big so setting to 36px is the same amount of the icon's height
}
```

Also change `.sb-slideover__close-button` top from 23px to 20px. What do you think?